providers/evercrypt/fst/EverCrypt.HKDF.fst

```diff
@@ -20,18 +20,21 @@ open Spec.Hash.Definitions
 open Spec.HKDF
 friend Spec.HKDF
 
+// TODO: Is this safe here? Can we safely break abstraction at this level?
+friend Lib.IntTypes
```

Comment:
I believe this is safe. This is no different than untrusted C code calling into verified code using `Lib.IntTypes` and enforcing secrecy internally.

Reply:
Comment removed.